client/src/pages/Home.js

```diff
@@ -17,7 +17,14 @@ function Home() {
 
   async function handleSubmit(event) {
     event.preventDefault();
-    const data = { name };
+    const currentDate = new Date();
+    const date =
```

Comment:
I think you could try "date.tolocalestring()" (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) to format you date, so you don't have to do it manually.

Reply:
Very nice :) Thanks. Already changed it :)